sdk/opentelemetry-sdk.gemspec

```
@@ -25,7 +25,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
   spec.required_ruby_version = '>= 2.5.0'
 
-  spec.add_dependency 'opentelemetry-api', '~> 1.0.0.rc3'
+  spec.add_dependency 'opentelemetry-api', '~> 1.0'
   spec.add_dependency 'opentelemetry-common', '~> 0.19.1'
```

```suggestion
  spec.add_dependency 'opentelemetry-common', '~> 0.19.2'
```
This is updated in this release.

---

Fixed this and the following similar comments in 4458d96c7bf17804d9008bb863535ec51c4781a2